app/services/api/registryService.js

```diff
@@ -35,6 +35,67 @@ angular.module('portainer.services')
     return deferred.promise;
   };
 
+  service.catalog = function(id) {
+    var deferred = $q.defer();
+
+    RegistryCatalog.get({id: id, limit: 200, last: ''}).$promise
+    .then(function success(data) {
+      var catalog = new RegistryCatalogViewModel(data.data, data.headers);
```

Comment:
Where are the files containing the models?

Reply:
Here : https://github.com/portainer/portainer/pull/1482/commits/2e5c3abb26c001501b50240da13c362b019bbcc4 😄